safety_down_under/content/_index.md

```diff
@@ -2,6 +2,6 @@
 title: 'Home'
 date: 2018-02-12T15:37:57+07:00
 heroHeading: 'Safety Down Under, Inc.'
-heroSubHeading: 'CPSI: Certified Playground Safety Inspectors. We provide playground inspections, maintenance, repair, installations, and engineered wood fiber safety surfacing.'
+heroSubHeading: 'Playground Safety Specialists. Certified WBENC Women's Business Enterprise. Certified DBE Disadvantaged Business Enterprise. Providing playground safety surfacing, playground inspections, playground maintenance, playground design and installation, gymnasium inspections and maintenance, fencing, and bulk landscape supplies.'
```

Comment:
change the enclosing single quotes to double quotes so that the apostrophe in `women's` doesn't get interpreted as the end of the subheading. i.e.

```suggestion
heroSubHeading: "Playground Safety Specialists. Certified WBENC Women's Business Enterprise. Certified DBE Disadvantaged Business Enterprise. Providing playground safety surfacing, playground inspections, playground maintenance, playground design and installation, gymnasium inspections and maintenance, fencing, and bulk landscape supplies."
```